### ISSUE TITLE
[Interaction Tutorial] Eliom_service.attach_post missing in API ?

### ISSUE THREAD

**Issue by yannham** [ocsigen/tutorial#74]
In the interaction tutorial, in the part dedicated to implement a user system, the tutorial makes use of `Eliom_service.attach_post` to create the connection service. The compiler (OCaml 4.04.2 with Eliom 6.2) can't find this symbol, and I didn't find any trace of a `attach_post` function in the code base or in recent and old APIs of Eliom. Am I missing something, or should `attach_post` be replaced by `create_attached_post` ?

**Comment by vasilisp** [comment]
Right, it should be replaced by `create_attached_post`. That function was originally called `attach_post` in the development version that led to Eliom 6.x, and apparently we missed an instance of it in the doc.

**Comment by yannham** [comment]
Ok, thanks for clarifying. It is worth noting that they are 4 occurrences of `attach_post` to substitute, and one of `attach_get`.
